features/step_definitions/town_steps.rb

```diff
@@ -20,9 +20,15 @@
 end
 
 
+Given(/^my non-feature branches are configured as (.*)$/) do |data|
+  set_configuration 'non-feature-branch-names',
+                    Kappamaki.from_sentence(data).join(', ')
+end
+
+
 Given(/^my perennial branches are configured as (.*)$/) do |data|
-  perennial_branches = Kappamaki.from_sentence(data).join(', ')
-  set_configuration 'perennial-branch-names', perennial_branches
+  set_configuration 'perennial-branch-names',
+                    Kappamaki.from_sentence(data).join(', ')
```

Comment:
Auhh stylistic differences. I prefer the old way as the function call stays on a single line.